website/docs/getting-started/using.md

```diff
@@ -2,16 +2,18 @@
 title: Using
 ---
 
-To use the Hook, first call it, telling it how many items your menu will have:
+To use the Hook, first call it, telling it how many items your menu will have along with a valid HTMLElement subtype:
 
 ```jsx
-const { buttonProps, itemProps, isOpen } = useDropdownMenu(numberOfItems);
+const { buttonProps, itemProps, isOpen } = useDropdownMenu<HTMLSubtypeElement>(numberOfItems);
```

Comment:
Since `HTMLSubtypeElement` is not a real type, and this documentation should reflect real usage I think this should either be changed to `HTMLElement`, or a common use-case like `HTMLButtonElement`